loki/transform/dependency_transform.py

```diff
@@ -319,7 +319,7 @@ def rename_imports(self, source, imports, targets=None):
         for im in imports:
             if im.c_import:
                 target_symbol = im.module.split('.')[0].lower()
-                if targets and target_symbol.lower() in targets:
+                if targets and target_symbol.lower() in targets and 'intfb' in im.module.lower():
```

Comment:
Are we already relying on `.intfb.h` as a naming convention for these imports? Otherwise we're implying a new restriction here.

Reply:
Yes we are relying on subroutine interface imports being named with an `.intfb.h` suffix. That felt like a safer assumption rather than stripping any c-style import containing the subroutine name.

Reply:
Yeah, I don't like hard-baking IFS conventions into this either, but we already have this in place at the moment, unfortunately. @reuterbal Maybe we should file a separate issue to un-hook this assumption from the `DependencyTransformation` and write a custom "IFS-specific" transformation that deals with those and makes the import chain coherent with a more generic import mechanism?